### ISSUE TITLE
Update accessibility overview so it can be used to measure compliance support

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#4205]
We need to update this overview:
https://github.com/digdir/designsystemet/wiki/Designsystemet-hjelper-deg-%C3%A5-oppfylla-WCAG%E2%80%90kravane

The goal is for the page to be used as a tool to measure which accessibility requirements Designsystemet supports in relation to compliance with universal design / accessibility regulations. This page will be linked from the benefit realization plan (Gevinstrealiseringsplan) and therefore needs to be accurate and clear.

- The list should clearly show which requirements in the regulations Designsystemet supports, and how it contributes to meeting them.
- The structure should make it easy for users to understand which requirements are relevant and which are not.

**Comment by frankdahle** [comment]
Draft is ready for some collaborative work. Meeting invite coming up!
[Designsystemet hjelper deg å oppfylle WCAG‐kravene · digdir/designsystemet Wiki](https://github.com/digdir/designsystemet/wiki/Designsystemet-hjelper-deg-%C3%A5-oppfylle-WCAG%E2%80%90kravene)

**Comment by mrosvik** [comment]
When this is finished, it should be published as part of this accessibility article:
https://designsystemet.no/no/fundamentals/introduction/accessibility

**Comment by Barsnes** [comment]
Add another level about patterns and AAA before closing

**Comment by eirikbacker** [comment]
Texts for inspiration in Mattilsynet: https://design.mattilsynet.no/?path=/docs/designsystem-introduksjon--docs#universell-utforming :)

**Comment by frankdahle** [comment]
New version of article is ready for review (Norwegian only). I've added the lists from Mattilsynet (recommended by Eirik) + a link to our Wiki overview of all WCAG criteras. I'll make the English version once it's approved.
https://github.com/digdir/designsystemet/blob/update-accessibility-article/apps/www/app/content/intro/no/accessibility.mdx
